package.json

```diff
@@ -29,6 +29,7 @@
         "i18next-http-backend": "^3.0.2",
         "linkify-react": "^4.2.0",
         "motion": "^12.4.7",
+        "qrcode.react": "^4.2.0",
```

Comment:
oop `react-qr-code` is already installed from the PaperWallet stuff, not sure which is better between the two, but can we just use one or the other? :D 🙏🏼